### DISCUSSION TITLE
Radeon graphic issue only with multiple screen LTSP debian Bullseye

### DISCUSSION THREAD

**Discussion by jivef** [ltsp/ltsp#687] [Q&A]
Hello,
I have experimented LTSP on PC without problem. It's easy. I use an old PC Lenovo, and it is OK.
I used a VM image with XFCE4. 

But I attempted to recycle an old "HP thin client T520 Flexible" with Radeon graphic card and I have an issue with this configuration.
I Have 2 output Diplay Port and 1 in VGA Port. 
I can use only one screen with this kind of machine.
When I connect 2 monitor, only one is working. 
I use the same VM image for "X86_64.img" PXE file.
I made another test with a M2 device card, and I installed debian Bullseye on the Thin Client and it's working correctly, in this configuration I can use multiple monitors. 
I don't know why, with my LTSP PXE image it doesn't work.

I didn't find the difference between the two configurations.
Have you ever encounter this king of problem ?
Do you know a solution ?

Thank you.
Best regards.
Jonas.

**Comment by alkisg** [comment]
Clone the Debian Bulleye installation in /srv/ltsp/x86_64 and run `ltsp image` and use that one to boot the client.
The "graphics working / non working" part is usually unrelated to LTSP, and it's caused by missing packages like drivers or firmware, or different kernel/xorg versions.

**Reply by jivef** [reply]
Hi,
Thank you for your answer.
I spent sometime to search the solution.
I delete some radeon package on the PXE VM template 
I remove and install again "firmware-amd-graphics" on the PXE template VM, and I install the same package on the LTSP server.
I run again   "ltsp initd" and "ltsp kernel" on the server.
And now it's OK.
Thank you.
